### PR TITLE
kubeadm: Fix some documentation errors

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This PR fixes some godoc typos

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @neolit123 @fabriziopandini

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/assign @rosti

File: cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ type NodeRegistrationOptions struct {
 
 	// Taints specifies the taints the Node API object should be registered with. If this field is unset, i.e. nil, in the `kubeadm init` process
 	// it will be defaulted to []v1.Taint{'node-role.kubernetes.io/master=""'}. If you don't want to taint your control-plane node, set this field to an
-	// empty slice, i.e. `taints: {}` in the YAML file. This field is solely used for Node registration.
+	// empty slice, i.e. `taints: []` in the YAML file. This field is solely used for Node registration.
```
Comment (fabriziopandini) [inline cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go]: If I'm not mistaken this fix can apply to the internal API as well...

**Review by fabriziopandini — APPROVED** [review]
Thanks @chuckha !
/approve
Could you kindly apply the same fix to the internal API as well?

File: cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go (hunk 1 above)
Comment (rosti) [reply cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go]: Indeed, let's have it fixed in the internal type too.

**Review by rosti — COMMENTED** [review]
Thanks @chuckha !

**Comment by chuckha** [comment]
@rosti ok! Updated the internal type as well

**Review by fabriziopandini — APPROVED** [review]
/lgtm
/approve

**Pull request merged**